### ISSUE TITLE
Excalidraw component steals focus

### ISSUE THREAD

**Issue by gaearon** [excalidraw/excalidraw#3685]
Standalone Excalidraw component tries to focus itself on page load. This leads to jumps on long pages. This seems like it should be configurable or off for the standalone version.

https://github.com/excalidraw/excalidraw/blob/abfc58eb24ee37af9f8a0922f7bf61f83d625aca/src/components/App.tsx#L836

My temporary workaround:

```js
if (typeof HTMLDivElement !== 'undefined') {
  const realFocus = HTMLDivElement.prototype.focus;
  HTMLDivElement.prototype.focus = function() {
    if (this.className.indexOf('excalidraw-container') === -1) {
      realFocus.apply(this, arguments);
    }
  }
}
```

**Comment by h7y** [comment]
Seems reasonable to disable it on standalone instances.

**Comment by ad1992** [comment]
Yeah, additionally it impacts accessibility as well, we can add `autofocus` prop and turn it off by default, consumers wanting to focus can turn it on.

**Comment by dwelle** [comment]
Let's do it.

**Comment by h7y** [comment]
I'll take this up?

**Comment by ad1992** [comment]
Sure go for it @h7y 👍🏻

**Comment by ad1992** [comment]
closed via #3691
